src/seedcase_sprout/core/path_local.py

```diff
@@ -0,0 +1,290 @@
+    """Gets the absolute path to the specified package's properties file.
+
+    Args:
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        The absolute path to the data package's properties file.
+
+    Examples:
+        ```{python}
+        import os
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+            # Create a package structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            sp.path_properties(path = temp_path)
+        ```
+    """
+    path = path / "datapackage.json"
+    return check_is_file(path)
+
+
+def path_readme(path: Path = Path.cwd) -> Path:
+    """Get the path to the README file for the data package.
+
+    Args:
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        The absolute path to the data package's README file.
+
+    Examples:
+        ```{python}
+        import os
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a package structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            sp.path_readme(path = temp_path)
+        ```
+    """
+    path = path / "README.md"
+    return check_is_file(path)
+
+
+def path_resource(resource_id: int, path: Path = Path.cwd()) -> Path:
+    """Gets the absolute path to the specified resource.
+
+    Args:
+        resource_id: The ID of the resource.
+        path: Provide a path to the package directory. Defaults to the current working
+            directory.
+
+    Returns:
+        The absolute path to a specific resource in the data package.
+
+    Examples:
+        ```{python}
+        import tempfile
+        from pathlib import Path
+
+        import seedcase_sprout.core as sp
+
+        # Create a temporary directory for the example
+        with tempfile.TemporaryDirectory() as temp_dir:
+            temp_path = Path(temp_dir)
+
+            # Create a package and resource structure first
+            sp.create_package_properties(
+                properties=sp.example_package_properties(),
+                path=temp_path
+            )
+
+            resource_path = Path(temp_path / "resources")
+            resource_path.mkdir()
+            sp.create_resource_structure(path=resource_path)
```

Comment:
```suggestion
            resources_path = Path(temp_path / "resources")
            resources_path.mkdir()
            sp.create_resource_structure(path=resources_path)
```
To avoid confusion with `path_resource`